app/views/admin/admin_users/_user_creations_summary.html.erb

```diff
@@ -1,17 +1,20 @@
-<% unless @works.empty? %>
+<% unless @user.works.empty? %>
+  <%# Checking @user.items.empty? rather than @items.empty? allows us to display
+# an empty listbox with pagination if the admin manually enters the wrong URL.
+# This is consistent with pagination on other site pages. %>
   <div class="work listbox group" id="works-summary">
-    <h3 class="heading"><%= t(".works", count: works.size) %></h3>
+    <h3 class="heading"><%= search_header(@works, nil, "Work") %></h3>
```

Comment:
We just call `pluralize` on the string passed here, which is... oof. A problem for future us.

https://github.com/otwcode/otwarchive/blob/3f1a3f96c02d9424378f98e8cfc7b460d50769d1/app/helpers/search_helper.rb#L4-L37

Reply:
:scream: